Improve phrasing in Iterator.prototype.take() (#37389)
mdn/content: files/en-us/web/javascript/reference/global_objects/iterator/take/index.md

// 5
```

> Because `fibonacci()` is an infinite iterator, using a `for` loop to iterate it without any logic to exit early (e.g. using a {{jsxref("Statements/break", "break")}} statement) would result in an infinite loop.

Reviewer: ```suggestion
Because `fibonacci()` is an infinite iterator, using a `for` loop to iterate it without any logic to exit early (such as a {{jsxref("Statements/break", "break")}} statement) would result in an infinite loop.
```
